app/api/views/briefs.py

```diff
@@ -625,27 +626,28 @@ def update_brief(brief_id):
         ):
             return forbidden('Unauthorised to edit drafts')
 
+    data_to_validate = copy.deepcopy(data)
```

Comment:
This is a general bug fix which was encountered while building document editing and editing validation - the validators actually cause a slight mutation on the data while cleaning it up, which means the brief became invalid after being published. By creating a deep copy of the data being validated, this no longer an issue.